halo2-ecc/src/grumpkin/mod.rs

```diff
@@ -0,0 +1,11 @@
+use crate::ecc;
+use crate::fields::fp;
+use crate::fields::native_fp;
+use crate::halo2_proofs::halo2curves::grumpkin::{Fq, Fr};
+
+pub type FpChip<'range, F> = native_fp::NativeFieldChip<'range, F>;
```

Comment:
the naming might feel weird

Reply:
might be better to avoid having this type alias